RecoLocalCalo/HGCalRecProducers/plugins/HGCalCLUEAlgo.cc

```diff
@@ -44,8 +44,13 @@ void HGCalCLUEAlgoT<T>::populate(const HGCRecHitCollection& hits) {
     if (dependSensor_) {
       int thickness_index = rhtools_.getSiThickIndex(detid);
       if (thickness_index == -1)
-        thickness_index = 3;
+        thickness_index = 6;
```

Comment:
can this also use `maxNumberOfThickIndices`?

Reply:
yes, maybe it is better to put it in the fillPSetDescription and avoid this hard-coded value.